src/main/java/edu/greenblitz/tobyDetermined/commands/rotatingBelly/FullRotateToPlace.java

```diff
@@ -0,0 +1,10 @@
+package edu.greenblitz.tobyDetermined.commands.rotatingBelly;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+public class FullRotateToPlace extends SequentialCommandGroup {
+    public FullRotateToPlace (){
+        addCommands(new RotateOutOfDoor());
```

Comment:
ParallelDeadlineGroup(new RotateOutOfDoor(), new WaitCommand(1));

Reply:
that's not what it should do. it should rotate backwards for 1 sec, then rotate until macroswitch